features/steps/promotion_steps.py

```diff
@@ -9,7 +9,7 @@
 from selenium.webdriver.support import expected_conditions
 from selenium.webdriver.support.ui import WebDriverWait, Select
 
-WAIT_SECONDS = int(getenv('WAIT_SECONDS', '10'))
+WAIT_SECONDS = int(getenv('WAIT_SECONDS', '480'))
```

Comment:
since this is not solving the bug, maybe we don't need to include this change?

Reply:
Yeah, changing this just for making sure whether it can fix the bug or not.